pkg/harvester-manager/machine-config/harvester.vue

```diff
@@ -729,7 +729,7 @@ export default {
         <transition-group>
           <div
             v-for="(disk, i) in disks"
-            :key="i"
+            :key="disk"
```

Comment:
This causes a different console warning (avoid using non-primitive values for keys) because `disk` is an object. I think you can use `${disk.bootOrder}${i}`

Reply:
Fixed